components/Tabs/Tabs.js

```diff
@@ -1,21 +1,26 @@
 class TabLink {
   constructor(tabElement){
     // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
+    this.tabElement= tabElement;
     
     // Get the `data-tab` value from this.tabElement and store it here
-    // this.tabData = ; 
+    this.tabData =  this.tabElement.dataset.tab; 
+    console.log(this.tabData); 
     
     // We need to find out if a user clicked 'all' cards or a specific category.  Follow the instructions below to accomplish this task:    
     
-    /* <- Delete this comment block when you work on the if statement
     // Check to see if this.tabData is equal to 'all'
-    if(){
+    if(this.tabData === "all"){
       // If `all` is true, select all cards regardless of their data attribute values
-      // this.cards = ;
+      console.log('it works');
```

Comment:
Great! another way to write an if/else is a ternary operator. It's cool. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator